From 959b570398445d6b350f1d97b963ef1bdaaeadff Mon Sep 17 00:00:00 2001
From: Dinesh Kumar Mirche <dmirche@habana.ai>
Date: Tue, 10 Jun 2025 09:06:19 +0300
Subject: [PATCH] [SW-230424] g3: sec: make register privilege

This patch makes the PHY_SIG_DETECT_OVRD register
in NIC_MAC_AUX from non privilege to privilege.
This is done to protect NIC ports disable/enable
functionality.

Change-Id: I50b84fec29e7075f61cc50b3ad7344beb471d011
Signed-off-by: Dinesh Kumar Mirche <dmirche@habana.ai>
---

diff --git a/drivers/misc/hl_security/gaudi3/gaudi3_protbits_configs.h b/drivers/misc/hl_security/gaudi3/gaudi3_protbits_configs.h
index 6f2ce3b..682505f 100644
--- a/drivers/misc/hl_security/gaudi3/gaudi3_protbits_configs.h
+++ b/drivers/misc/hl_security/gaudi3/gaudi3_protbits_configs.h
@@ -172,7 +172,7 @@
 0x3ef }
 
 #define GAUDI3_PROTBITS_DATA462 { \
-0x3f }
+0x2f }
 
 #define GAUDI3_PROTBITS_DATA463 { \
 	0xfffff3f1, 0x1fff }
